docs/cloud/features/test-replay.mdx

```diff
@@ -114,6 +132,111 @@ Toggle each test attempt, play back errors at a variety of speeds or scrub throu
   alt="Test Replay scrubber controls"
 />
 
+## How Test Replay Works
+
+Our aim is to create an impactful debugging experience that covers the most ground for users.
+Cypress captures a wide range of data during end-to-end and component test runs.
+This data is then uploaded and stored in Cypress Cloud and replayed for debugging through Test Replay.
+
+### What is captured
+
+- Test commands
+- Network requests
+- Console logs
+- JavaScript errors
+- Element rendering
+- DOM mutations
+- CSS styles
+- SVG elements
+- Iframes
+- Shadow DOM
+- Canvas elements (v13.5.0+, please contact us at support@cypress.io to enable capturing canvas elements)
+
+### What is not captured
+
+Currently Test Replay does not support the following:
+
+- Tests recorded in Cypress App below v13.
+- WebKit browser (Safari) ([see issue](https://github.com/cypress-io/cypress/issues/28895))
+- Firefox browser ([see issue](https://github.com/cypress-io/cypress/issues/28894))
+- Video ([see issue](https://github.com/cypress-io/cypress/issues/29903))
+- Audio ([see issue](https://github.com/cypress-io/cypress/issues/29905))
+- Objects that do not have `type="image/svg+xml"` ([see issue](https://github.com/cypress-io/cypress/issues/29904))
+- Shadow DOM with `slotAssignment="manual"` ([see issue](https://github.com/cypress-io/cypress/issues/29107))
+- Shadow DOM canvas ([see issue](https://github.com/cypress-io/cypress/issues/30438))
+- Server sent events ([see issue](https://github.com/cypress-io/cypress/issues/29908))
+- Web sockets ([see issue](https://github.com/cypress-io/cypress/issues/29907))
+- localStorage & sessionStorage ([see issue](https://github.com/cypress-io/cypress/issues/29909))
+- Cookies ([see issue](https://github.com/cypress-io/cypress/issues/29910))
+- Console props of Cypress commands outside of `cy.request()` ([see issue](https://github.com/cypress-io/cypress/issues/29911))
+
+### Performance
+
+#### Recording
+
+Capturing this additional data can use additional resources while recording, but you can
+expect an even performance exchange with Test Replay versus video recording.
+There is more activity to capture Test Replay data but without the overhead of recording,
+compressing, and storing video assets.
+
+:::info
+
+**Tip:** We recommend disabling video recording when recording with
```

Comment:
is it clear that video is disabled by default?

Reply:
There are so many customers I've seen that have it on, I do think it needs to be called out very explicitly.